dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java

```diff
@@ -809,4 +814,10 @@ private Constants() {
      * tenant
      */
     public static final int TENANT_FULL_NAME_MAX_LENGTH = 30;
+
+    /**
+     * schedule time
+     */
+    public static final int SCHEDULE_TIME_MAX_LENGTH = 100;
```

Comment:
Maybe we should add some docs about why we should set this value to `100` instead of `200` or other values

Reply:
This figure is neither too much nor too little. It is tentatively set at 100 at present